README.md

```diff
@@ -54,6 +63,403 @@ npm install @dojo/shim
 
 The examples below are provided in TypeScript syntax. The package does work under JavaScript, but for clarity, the examples will only include one syntax.
 
+### Widget Routing
+
+Widgets are a fundamental concept for any Dojo 2 application and as such Dojo 2 Routing provides collection of components that integrate directly with existing widgets within an application.
+
+These components enable widgets to be registered against a route _without_ requiring any knowledge of the `Router` or `Routes`.
+
+#### Outlets
+
+The primary concept for the routing integration is an `outlet`, this is a unique identifier associated with registered application route. Dojo 2 Widgets can then be configured with these outlet identifiers using the `Outlet` higher order component. `Outlet` returns a "new" Widget that can be used like any other widget within a `render` method e.g. `w(MyFooOutlet, { })`.
+
+Properties can be passed to an `Outlet` widget, in the same way as if the original widget was being used, however all properties are made optional in order to allow the properties to be injected using the [mapParams](#mapParams) function described below.
```

Comment:
"widget, in the same way as if the original widget was being used, however all properties are made optional in order to allow the properties to be injected" -> "widget in the same way as if the original widget was being used. However, all properties are made optional to allow the properties to be injected"